packages/vendure-scripts/README.md

```diff
@@ -0,0 +1,17 @@
+# Vendure Script
+
+## [Official documentation here](https://pinelab-plugins.com/plugin/vendure-scripts)
+
+This 'plugin' contains generic scripts that can be used with `Vendure` projects.
+
+## Getting started
```

Comment:
Can you add a separate section for `Assign all products to channel`, with a short description about what this script does and what its for?